app/src/main/java/ch/epfl/culturequest/social/User.java

```diff
@@ -0,0 +1,13 @@
+package ch.epfl.culturequest.social;
+
+import com.google.firebase.auth.FirebaseAuthProvider;
+import com.google.firebase.auth.FirebaseUser;
+
+public interface User {
+    String getDisplayName();
+    String getFirstName();
+    String getLastName();
+    String getUid();
+    String getEmail();
+    String getPhoneNumber();
+}
```

Comment:
I don't think that this interface is necessary since we don't plan on having different kind of user for the moment